jekyll/_cci2/docker-layer-caching.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: classic-docs
+title: "Docker Layer Caching"
+short-title: "Docker Layer Caching"
+order: 5
+---
+
+{% include beta-premium-feature.html%}
```

Comment:
In this call we should add `feature='Docker Layer Caching'`, so that label appears in the banner.